CMakeLists.txt

```diff
@@ -12,3 +12,8 @@ install(DIRECTORY launch
 install(PROGRAMS scripts/hydro-upgrade-notice
   DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}/
 )
+
+if(CATKIN_ENABLE_TESTING)
+  find_package(roslaunch REQUIRED)
+  roslaunch_add_file_check(test/camera_for_test.launch)
```

Comment:
+1 for `roslaunch_add_file_check`. 
Why not check the entire `test`, and `launch` folders?